pkg/build/linux.go

```diff
@@ -216,6 +223,20 @@ func kernelBin(arch string) string {
 	}
 }
 
+var linuxCompilerRegexp = regexp.MustCompile("#define\\s+LINUX_COMPILER\\s+\"(.*)\"")
```

Comment:
Regexp's are generally easier to express using raw string literals (with ticks), then you don't need to repeat \ char, can use " as necessary, etc.

Reply:
Done